backend/cfg/cfgize.ml

```diff
@@ -645,7 +645,7 @@ module Trap_depth_and_exns = struct
       block.trap_depth <- succ (List.length stack);
       (* map from handlers reachable from the block to stacks at the start of
          such blocks; used to both know which other blocks should be visited (as
-         exceptional successors) and to populate the `exns` field. *)
+         exceptional successors) and to populate the `exn` field. *)
       let table = Label.Tbl.create 17 in
       let stack =
```

Comment:
I think this can also be simplified.

Reply:
Quite probably, I was more or less going for the shortest
diff. I will have another look.

Reply:
Tentative simplification in https://github.com/ocaml-flambda/flambda-backend/pull/470/commits/2c41fc9a584015644270344266f0cc22561e2633.

Reply:
It looks good! I just have a small suggestion.

- I think it may be simpler to understand the code if `exceptional_successor` is not a `ref`. Instead, the transformers `terminator` and `basic` and `record_handler` can return a pair (stack, exceptional_successor)., and I guess basic would have to take a pair of the same type for fold.

- function  names `terminator` and `basic` sound like they return an instruction, it would be better to name them something like `process_terminator` and `process_basic`.

- It would be useful to have a comment for `handler_option` explaining what it is how it is used (instead of the obsolete comment you've removed from `update_blocks`).